bin/p2-inspect/main.go

```diff
@@ -81,7 +81,7 @@ func main() {
 			}
 
 			old := statusMap[podId][node]
-			_, old.Health = health.FindWorst(results)
+			_, old.Health = health.GetMultisourceResult(results)
```

Comment:
I doesn't need to. I just thought the name FindWorst didn't represent super accurately what the method would do.